theme/azul.js

```diff
@@ -0,0 +1,209 @@
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+(function (root, factory) {if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define(['exports', 'echarts'], factory);
+    } else if (typeof exports === 'object' && typeof exports.nodeName !== 'string') {
+        // CommonJS
+        factory(exports, require('echarts'));
+    } else {
+        // Browser globals
+        factory({}, root.echarts);
+    }
+}(this, function (exports, echarts) {
+    var log = function (msg) {
+        if (typeof console !== 'undefined') {
+            console && console.error && console.error(msg);
+        }
+    };
+    if (!echarts) {
+        log('ECharts is not Loaded');
+        return;
+    }
+
+    var colorPalette = [
+        '#f2385a','#f5a503','#e6efdc','#4ad9d9',
+        '#e6efdc','#f7879c','#fccfd7','#d5f6f6'
+    ];
+
+    var theme = {
+
+        color: colorPalette,
+
+        title: {
+            textStyle: {
+                fontWeight: 'normal',
+				color: '#f2385a'
+            }
+        },
+
+        visualMap: {
+            color:['#f2385a','#f5a503']
+        }, 
+
+        toolbox: {
+            color : ['#f2385a','#f2385a','#f2385a','#f2385a']
+        },
+
+    tooltip: {
+        backgroundColor: 'rgba(0,0,0,0.5)',
+        axisPointer : {            // Axis indicator, coordinate trigger effective
+            type : 'line',         // The default is a straight line： 'line' | 'shadow'
+            lineStyle : {          // Straight line indicator style settings
+                color: '#f2385a',
+                type: 'dashed'
+            },
+            crossStyle: {
+                color: '#f2385a'
+            },
+            shadowStyle : {                     // Shadow indicator style settings
+                color: 'rgba(200,200,200,0.3)'
+            }
+        }
+    },
+
+    // Area scaling controller
+    dataZoom: {
+        dataBackgroundColor: '#eee',            // Data background color
+        fillerColor: 'rgba(200,200,200,0.2)',   // Fill the color
+        handleColor: '#f2385a'     // Handle color
+    },
+
+    timeline : {
+        lineStyle : {
+            color : '#f2385a'
+        },
+        controlStyle : {
+            normal : { color : '#f2385a'},
+            emphasis : { color : '#f2385a'}
+        }
+    },
+
+	k: {
+        itemStyle: {
+            normal: {
+                color: '#f5a503',          // Yang line filled with color
+                color0: '#e6efdc',      // Yinxian fill color
+                lineStyle: {
+                    width: 1,
+                    color: '#e6efdc',   // Yang line border color
+                    color0: '#f5a503'   // Yinbian border color
+                }
+            }
+        }
```

Comment:
The hierarchy of `itemStyle` since 4.0.0 has been changed to

```js
itemStyle: {
   color: ...
},
lineStyle: {
   color: ...
},
areaStyle: {
   color: ...
}
```

Reply:
I'm a little confused about this. Please see post on main thread. Thanks!